index.js

```diff
@@ -59,7 +59,7 @@ app.use((error, req, res, next) => {
     res.status(500).send();
   });
 // set port
-app.listen(3010, function() {
+app.listen(8080, function() {
     console.log('Node app is running on port 3010');
```

Comment:
Shall we change this comment as well ? :)

Reply:
Added a todo as we don't have proper logger as for now.